lib/omniauth/strategies/openid_connect.rb

```diff
@@ -120,6 +121,9 @@ def callback_phase
 
         discover!
         client.redirect_uri = redirect_uri
+
+        return id_token_callback_phase unless options.response_type.to_s == 'code'
```

Comment:
Please consider changing check to straight logic, instead of reverted

```ruby
return id_token_callback_phase if options.response_type.to_s == 'id_token'
```

Reply:
I tried both solutions and to be honest I don't have a strong opinion on this one.
The reason why I chose to check for the `code` value is because the rest of the flow depends on the resnpose_type code. If you prefer to stick to `id_token` as the condition for return, I'll change it in next commit.

Reply:
This could be a problem if we decide to add more types later.

Reply:
✅